core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -45,6 +45,7 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->children()
                 ->scalarNode('web_dir')
+                    ->info('Absolute path to the web directory.')
```

Comment:
Suggestion: *Absolute path to the web directory. Make sure to use the `%kernel.project_dir%` parameter for the absolute path prefix.*